src/radical/pilot/agent/launch_method/flux.py

```diff
@@ -15,44 +15,64 @@ class Flux(LaunchMethod):
     #
     def __init__(self, name, lm_cfg, rm_info, session, prof):
 
-        LaunchMethod.__init__(self, name, lm_cfg, rm_info, session, prof)
+        self._flux_handles = list()
+        self._details      = list()
 
+        super().__init__(name, lm_cfg, rm_info, session, prof)
 
     # --------------------------------------------------------------------------
     #
     def _terminate(self):
 
-        if self._fh:
-            self._fh.reset()
+        for fh in self._flux_handles:
+            fh.reset()
 
 
     # --------------------------------------------------------------------------
     #
     def _init_from_scratch(self, env, env_sh):
 
-        self._prof.prof('flux_start')
-        self._fh = ru.FluxHelper()
 
-        self._log.debug('starting flux')
+        n_partitions        = self._rm_info.details.get('n_partitions', 1)
+        n_nodes             = len(self._rm_info.node_list)
+        nodes_per_partition = int(n_nodes / n_partitions)
+        threads_per_node    = self._rm_info.cores_per_node * self._rm_info.threads_per_core
```

Comment:
should be `threads_per_node = self._rm_info.cores_per_node`, Cores in RMInfo are vCores (already consider number of threads)

Reply:
Thanks, fixed.